// web-app/src/features/recipes/api/mockData.js
export const mockRecipes = [
  {
    id: 1,
    title: "Pâtes à la carbonara",
    image: "https://images.radio-canada.ca/v1/alimentation/recette/16x9/pates-carbonara-71295.jpg",
    description: "Une recette italienne crémeuse et rapide à faire.",
    time: "15 min",
    calories: 550,
    moment: "soir",
    type: "chaud",
    tip: "Ajoutez une salade verte pour équilibrer le repas.",
    ingredients: [
      { name: "pâtes", quantity: 200, unit: "g", category: "féculent" },
      { name: "œufs", quantity: 1, unit: "", category: "protéine_animale" },
      { name: "parmesan", quantity: 25, unit: "g", category: "protéine_animale" },
      { name: "lardons", quantity: 50, unit: "g", category: "protéine_animale" }
    ],
    defaultServings: 2,

    steps: [
      "Faire bouillir l’eau.",
      "Cuire les pâtes pendant 10 min.",
      "Battre les œufs avec le parmesan.",
      "Ajouter les lardons à la poêle.",
      "Mélanger hors du feu avec les œufs.",
    ]
  },
   {
    id: 2,
    title: "Pâtes à la carbonara",
    image: "https://images.radio-canada.ca/v1/alimentation/recette/16x9/pates-carbonara-71295.jpg",
    description: "Une recette italienne crémeuse et rapide à faire.",
    time: "15 min",
    calories: 550,
    moment: "soir",
    type: "chaud",
    tip: "Ajoutez une salade verte pour équilibrer le repas.",
    ingredients: [
      { name: "pâtes", quantity: 200, unit: "g", category: "féculent" },
      { name: "œufs", quantity: 1, unit: "", category: "protéine_animale" },
      { name: "parmesan", quantity: 25, unit: "g", category: "protéine_animale" },
      { name: "lardons", quantity: 50, unit: "g", category: "protéine_animale" }
    ],
    defaultServings: 2,

    steps: [
      "Faire bouillir l’eau.",
      "Cuire les pâtes pendant 10 min.",
      "Battre les œufs avec le parmesan.",
      "Ajouter les lardons à la poêle.",
      "Mélanger hors du feu avec les œufs.",
    ]
  },
   {
    id: 3,
    title: "Pâtes à la carbonara",
    image: "https://images.radio-canada.ca/v1/alimentation/recette/16x9/pates-carbonara-71295.jpg",
    description: "Une recette italienne crémeuse et rapide à faire.",
    time: "15 min",
    calories: 550,
    moment: "soir",
    type: "chaud",
    tip: "Ajoutez une salade verte pour équilibrer le repas.",
    ingredients: [
      { name: "pâtes", quantity: 200, unit: "g", category: "féculent" },
      { name: "œufs", quantity: 1, unit: "", category: "protéine_animale" },
      { name: "parmesan", quantity: 25, unit: "g", category: "protéine_animale" },
      { name: "lardons", quantity: 50, unit: "g", category: "protéine_animale" }
    ],
    defaultServings: 2,

    steps: [
      "Faire bouillir l’eau.",
      "Cuire les pâtes pendant 10 min.",
      "Battre les œufs avec le parmesan.",
      "Ajouter les lardons à la poêle.",
      "Mélanger hors du feu avec les œufs.",
    ]
  },
   {
    id: 4,
    title: "Pâtes à la carbonara",
    image: "https://images.radio-canada.ca/v1/alimentation/recette/16x9/pates-carbonara-71295.jpg",
    description: "Une recette italienne crémeuse et rapide à faire.",
    time: "15 min",
    calories: 550,
    moment: "soir",
    type: "chaud",
    tip: "Ajoutez une salade verte pour équilibrer le repas.",
    ingredients: [
      { name: "pâtes", quantity: 200, unit: "g", category: "féculent" },
      { name: "œufs", quantity: 1, unit: "", category: "protéine_animale" },
      { name: "parmesan", quantity: 25, unit: "g", category: "protéine_animale" },
      { name: "lardons", quantity: 50, unit: "g", category: "protéine_animale" }
    ],
    defaultServings: 2,

    steps: [
      "Faire bouillir l’eau.",
      "Cuire les pâtes pendant 10 min.",
      "Battre les œufs avec le parmesan.",
      "Ajouter les lardons à la poêle.",
      "Mélanger hors du feu avec les œufs.",
    ]
  },
   {
    id: 5,
    title: "Pâtes à la carbonara",
    image: "https://images.radio-canada.ca/v1/alimentation/recette/16x9/pates-carbonara-71295.jpg",
    description: "Une recette italienne crémeuse et rapide à faire.",
    time: "15 min",
    calories: 550,
    moment: "soir",
    type: "chaud",
    tip: "Ajoutez une salade verte pour équilibrer le repas.",
    ingredients: [
      { name: "pâtes", quantity: 200, unit: "g", category: "féculent" },
      { name: "œufs", quantity: 1, unit: "", category: "protéine_animale" },
      { name: "parmesan", quantity: 25, unit: "g", category: "protéine_animale" },
      { name: "lardons", quantity: 50, unit: "g", category: "protéine_animale" }
    ],
    defaultServings: 2,

    steps: [
      "Faire bouillir l’eau.",
      "Cuire les pâtes pendant 10 min.",
      "Battre les œufs avec le parmesan.",
      "Ajouter les lardons à la poêle.",
      "Mélanger hors du feu avec les œufs.",
    ]
  }
];
